DataFormats/Detectors/CPV/src/Cluster.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DataFormatsCPV/Cluster.h"
+#include "CPVBase/Geometry.h"
+#include "CPVBase/CPVSimParams.h"
+
+using namespace o2::cpv;
+
+ClassImp(Cluster);
+
+//____________________________________________________________________________
+bool Cluster::operator<(const Cluster& other) const
+{
+  // Compares two Clusters according to their position in the CPV modules
+
+  char cpvmod1 = getModule();
+  char cpvmod2 = other.getModule();
+  if (cpvmod1 != cpvmod2) {
+    return cpvmod1 < cpvmod2;
+  }
+
+  float posX, posZ;
+  getLocalPosition(posX, posZ);
+  float posOtherX, posOtherZ;
+  other.getLocalPosition(posOtherX, posOtherZ);
+  int rowdifX = (int)std::ceil(posX / o2::cpv::CPVSimParams::Instance().mSortingDelta) -
+                (int)std::ceil(posOtherX / o2::cpv::CPVSimParams::Instance().mSortingDelta);
+  if (rowdifX == 0) {
+    return posZ > posOtherZ;
+  } else {
+    return rowdifX > 0;
+  }
+}
+//____________________________________________________________________________
+/// \brief Comparison oparator, based on time and absId
+/// \param another CPV Cluster
+/// \return result of comparison: x and z coordinates
+bool Cluster::operator>(const Cluster& other) const
+{
+  // Compares two Clusters according to their position in the CPV modules
+
+  if (mModule != other.mModule) {
+    return mModule > other.mModule;
+  }
+
+  int rowdifX = (int)std::ceil(mLocalPosX * o2::cpv::CPVSimParams::Instance().mSortingDelta) -
+                (int)std::ceil(other.mLocalPosX * o2::cpv::CPVSimParams::Instance().mSortingDelta);
```

Comment:
I mean here.

Reply:
thanks, fixed

Reply:
@peressounko I don't see it fixed: as far as I understand, the ``mSortingDelta`` is the *inverse* pitch, so here you get the rowID by multiplying the coordinate by it. But on the line https://github.com/AliceO2Group/AliceO2/blob/7159f3b14fe7aad55961d75e5204ca9a391fce22/DataFormats/Detectors/CPV/src/Cluster.cxx#L34 you obtain similar rowID by *dividing* by the same inverse pitch, you should also multiply there.

Reply:
This is some glich in git: I have in my local code
`  int rowdifX = (int)std::ceil(posX * o2::cpv::CPVSimParams::Instance().mSortingDelta) -
                (int)std::ceil(posOtherX * o2::cpv::CPVSimParams::Instance().mSortingDelta);`
changes are committed and pushed to github. But here I do not see new version...

Reply:
I think, in last version this is fixed

Reply:
indeed a glitch, now it shows corrected version...